src/status_im/signals/core.cljs

```diff
@@ -67,6 +67,8 @@
       "envelope.expired"   (transport.message/update-envelopes-status cofx (:ids (js->clj event-js :keywordize-keys true)) :not-sent)
       "message.delivered"  (let [{:keys [chatID messageID]} (js->clj event-js :keywordize-keys true)]
                              (models.message/update-db-message-status cofx chatID messageID :delivered))
+     ;; "message.seen"       (let [{:keys [chatID messageID]} (js->clj event-js :keywordize-keys true)]
```

Comment:
could you elaborate?

Reply:
Yeah! Actually I should have omitted this line of code from this PR. After having a consultation I came to understand that the back-end implementation to process the status of a message as having been seen is not yet implemented and so this code is not necessary until i implement the ability to send a message signal to process the message status as having been seen which would be on status-go